packages/experimental/ActivityIndicator/src/ActivityIndicator.tsx

```diff
@@ -82,7 +81,7 @@ export const ActivityIndicator = compose<ActivityIndicatorType>({
       const { ...mergedProps } = mergeProps(props, rest, otherRootProps);
       return (
         <Slots.root {...mergedProps}>
-          <Slots.svg {...otherSvgProps}>{path}</Slots.svg>
+          <Slots.svg {...animatedSvgProps}>{path}</Slots.svg>
```

Comment:
Don't we also need to merge this with `slotProps.svg` ? Which I suppose is just width/height

Reply:
No, because buildProps already adds those props within styling.tsx.

Reply:
yes, but to get those in here, we need to grad it from UseStyling(props).svg, aka `slotProps.svg`, do we not?

Reply:
No, the changes directly apply from .styling.tsx. I tried changing it to different heights and the changes show up. Also, you can see that we don't merge slotProps.root to get the accessibility and style to work on root either.

Reply:
Oh, interesting. Thanks for the clarification!